sass/style.scss

```diff
@@ -108,3 +108,36 @@ body
   border: 1px solid black !important;
   box-sizing: border-box;
 }
+.version-tag
```

Comment:
These styles are in the wrong place in the CSS file. At the top, we have an area for "all pages" which woul dbe a better fit for this.